src/matrix.rs

```diff
@@ -197,6 +198,26 @@ impl<S: BaseFloat> Matrix3<S> {
         Matrix3::from_cols(side, up, dir).transpose()
     }
 
+    /// Create a rotation matrix that will cause a vector to point at
+    /// `dir`, using `up` for orientation.
+    pub fn look_at_lh(dir: Vector3<S>, up: Vector3<S>) -> Matrix3<S> {
+        let dir = dir.normalize();
+        let side = up.cross(dir).normalize();
+        let up = dir.cross(side).normalize();
+
+        Matrix3::from_cols(side, up, dir).transpose()
+    }
+
+    /// Create a rotation matrix that will cause a vector to point at
+    /// `dir`, using `up` for orientation.
+    pub fn look_at_rh(dir: Vector3<S>, up: Vector3<S>) -> Matrix3<S> {
```

Comment:
there are 3 functions now that only differ in a single `-` sign. Can we have a helper method that all of 3 would call?

Reply:
@kvark  I've removed the duplication here.